ServerCore/Pages/Resources/Default/HomePartial.cshtml

```diff
@@ -1,24 +1,10 @@
-﻿@{
-    var eventPage = Model as ServerCore.ModelBases.EventSpecificPageModel;
-    var Event = eventPage?.Event;
+﻿<h2>@Model?.Event?.Name homepage TBD</h2>
+@if (Model?.Event?.EventRole == ModelBases.EventRole.admin)
+{
+    <p><b>Since you're an admin:</b> Upload a file named home-content.html to the <a href="https://puzzlehunt.azurewebsites.net/@(Model.Event.UrlString)/admin/Puzzles/ResourceManagement">shared resources folder</a> for this event. Don't forget the rules and FAQ, and for good measure, set the event's HomePartial field to be the same as its UrlString so this content is permanently hidden.</p>
```

Comment:
This message should clearly provide all filenames:

- faq-content.html
- home-content-html
- rules-content-html
- global-styles.css

Reply:
Also, new event creation should really provide these files by default instead of relying on this magic string property that acts like a bool. Then we could deprecate the HomePartial property as well.

Reply:
Given that I don't believe we need these files at all moving forward, I'm closing this conversation.  See the one above as to why.